Libplanet/Blocks/Block.cs

```diff
@@ -110,16 +107,15 @@ private Block(RawBlock rb)
             DateTimeOffset timestamp,
             IEnumerable<Transaction<T>> transactions)
         {
-            Transaction<T>[] orderedTxs = transactions.OrderBy(tx => tx.Nonce).ToArray();
+            var txs = transactions.OrderBy(tx => tx.Id).ToImmutableArray();
```

Comment:
We sort transactions in [line43](https://github.com/planetarium/libplanet/pull/511/files#diff-e971906424c095136a07b8eefbdb6511R43) again. Is this line required?

Reply:
Yes, it's because the process to order transactions determines `Nonce`. Isn't it?